### PR TITLE
Delete Assembly Hot Fix

### PR DESCRIPTION
**Pull request by Dhruv-0-Arora**
### Description
- [x] Fixed Bug: deleting assemblies from ManageAssembliesModal does not delete the transform gizmo attached
- [x] Removed duplicate GetRootNodeId() implementation in MirabufSceneObject.ts
- [x] Removed inaccessible variable _deleteGizmoOnEscape

### CODE REVIEW

File: fission/src/ui/modals/spawning/ManageAssembliesModal.tsx [hunk 1]
```diff
@@ -20,7 +20,9 @@ const AssemblyCard: React.FC<AssemblyCardProps> = ({ id, update }) => {
             <Label className="text-wrap break-all">{id}</Label>
             <Button
                 value="Delete"
+                // prettier-ignore
                 onClick={() => {
+                    (World.SceneRenderer.sceneObjects.get(id) as MirabufSceneObject).DisableTransformControls()
```
Comment (HunterBarclay) [inline fission/src/ui/modals/spawning/ManageAssembliesModal.tsx]: I would move this to be included in the Dispose function of `MirabufSceneObject`

File: fission/src/mirabuf/MirabufSceneObject.ts [hunk 2]
```diff
@@ -205,6 +196,9 @@ class MirabufSceneObject extends SceneObject {
         return mesh
     }
 
+    /**
+     * Changes the mode of the mirabuf object from being interacted with to being placed.
+     */
     public EnableTransformControls(): void {
```
Comment (HunterBarclay) [inline fission/src/mirabuf/MirabufSceneObject.ts:314]: What happens if this is called when the transform gizmo is already enabled? Will it handle it accordingly?

**Review by HunterBarclay — CHANGES_REQUESTED** [review]

**Review by a-crowell — COMMENTED** [review]
Hotfix looks good! Once `EnableTransformControls` handles already being enabled, LGTM.

**Review by a-crowell — APPROVED** [review]
Lovely

**Review by HunterBarclay — APPROVED** [review]

**Comment by HunterBarclay** [comment]
@Dhruv-0-Arora Please fix merge conflicts

**Comment by Dhruv-0-Arora** [comment]
> @Dhruv-0-Arora Please fix merge conflicts

Fixed them 👍

**Pull request merged**